src/ui/server/controllers/insurance/check-your-answers/policy/index.test.ts

```diff
@@ -159,6 +162,19 @@ describe('controllers/insurance/check-your-answers/policy', () => {
           expect(res.redirect).toHaveBeenCalledWith(PROBLEM_WITH_SERVICE);
         });
       });
+
```

Comment:
Can we add a test for calling the API? Seems it's missing for currencies as well (probably my fault, sorry)

```js
it('should call api.keystone.countries.getAll', async () => {
it('should call api.keystone.currencies.getAll', async () => {
``